tests/packagers/index.test.js

```diff
@@ -4,49 +4,28 @@
  */
 
 const _ = require('lodash');
-const chai = require('chai');
-const sinon = require('sinon');
-const mockery = require('mockery');
 const Serverless = require('serverless');
+const baseModule = require('../../lib/packagers/index');
 
-chai.use(require('sinon-chai'));
-
-const expect = chai.expect;
+jest.mock('fs-extra');
+jest.mock('../../lib/packagers/npm', () => {
+  return {
+    hello: 'I am NPM'
+  };
+});
+jest.mock('../../lib/packagers/yarn', () => ({
+  hello: 'I am Yarn'
+}));
```

Comment:
I want to make it look really similar, makes skimming easier when reading lots of codes at once.

```suggestion
jest.mock('../../lib/packagers/npm', () => ({
  hello: 'I am NPM'
}));
jest.mock('../../lib/packagers/yarn', () => ({
  hello: 'I am Yarn'
}));
```